docs/device-management-test.md

```diff
@@ -1,6 +1,6 @@
 <h2 id="device-management-test">Validation and testing</h2>
 
-Mbed Device Management provides built-in tests to help you when define your device management configuration. Before running these tests, we recommend you refer to the [testing setup](#testing-setup) section below. 
+Device Management provides built-in tests to help you when define your device management configuration. Before running these tests, we recommend you refer to the [testing setup](#testing-setup) section below.
```

Comment:
Any way to rephrase this so it's not repetitive?

Reply:
I think the "help you when define" is the real issue